CHANGELOG.md

```diff
@@ -12,6 +12,258 @@ The format is inspired by [Keep a Changelog].
 
 <!-- TOWNCRIER -->
 
+## 22.1 (2022-04-01)
+
+| Protocol          | Version   |
+|:------------------|:---------:|
+| Consensus         | 6.0.0     |
+| Runtime Host      | 5.0.0     |
+| Runtime Committee | 4.0.0     |
+
+### Removals and Breaking Changes
+
+- governance: Allow unregistered nodes in votes
+  ([#4607](https://github.com/oasisprotocol/oasis-core/issues/4607))
+
+  The intent from the comments is to allow votes to be cast as long as the
+  entity has at least 1 validator in the active set.  This fixes entities
+  with nodes that are not currently registered having their votes rejected.
+
+- go/consensus/tendermint: Record node liveness successes
+  ([#4615](https://github.com/oasisprotocol/oasis-core/issues/4615))
+
+- scheduler: Improve runtime committee election fairness
+  ([#4623](https://github.com/oasisprotocol/oasis-core/issues/4623))
+
+  Prune the per-entity nodes down to the per-runtime cap if configured,
+  prior to doing the election, to improve fairness when entities opt to
+  operate multiple nodes for the same runtime.
+
+### Configuration Changes
+
+- go/oasis-node/cmd/node: Do not allow running oasis-node as root
```

Comment:
Maybe this should be bolded and repeated in the Release notes/on top of the changelog.

Reply:
Updated, can you check if this looks better now?

Reply:
Yes, I think it is better now.